app/assets/javascripts/lib/views/dashboard_changer_view.js

```diff
@@ -103,18 +103,18 @@
    * Called upon successful completion of the XHR request.
    */
   DashboardChangerView.prototype.onDone = function (data, status, jqXHR) {
-    var constraintsEl  = $('#dashboard');
+    var dashboardItemsEl  = $('#dashboard');
```

Comment:
@noracato Would you mind taking care of this before the branch is merged?

If you're not aware, the no-undef rule is because ESLint expects global variables to be declared in a comment at the top of the file:

```js
/* globals $ _ App */
```

Reply:
I will :) Thanks for your explaination!